sonic-pcied/scripts/pcied

```diff
@@ -151,6 +160,28 @@ class DaemonPcied(daemon_base.DaemonBase):
 
         self.status_table.set("status", fvs)
 
+    # Check if any interface is in detaching mode by querying the state_db
```

Comment:
@vvolam interface or pci end point in this case the DPU?

Reply:
@prgeor Here I mean PCIE interface. And DPU here is one of the PCIE interface.